packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -130,6 +130,15 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     <>
       <div ref={messageRef}>
         <Chat.Message
+          aria-label={_formatString(strings.messageContentAriaLabel, {
+            displayName: `${message.mine ? strings.localSenderAriaLabel : message.senderDisplayName}`,
+            timeStamp: `${
+              showDate
+                ? formatTimestampForChatMessage(message.createdOn, new Date(), strings)
+                : formatTimeForChatMessage(message.createdOn)
+            }`,
+            content: `${message.content}`
```

Comment:
I'm worried content is not always a nice string, what if it contains a gif? or other html elements:
![image](https://user-images.githubusercontent.com/2684369/165164331-dfc091ad-2ffc-465c-ae18-10ab3ef16b91.png)

Reply:
added tostring() method

Reply:
Great catch. I hadn't thought of that at all. Perhaps we should only announce the content when type is `'text'` and instead announce a generic message for the other types: "A non-text message was sent by <display_name>".

Later on, perhaps we can start parsing html to get alt text and get fancier :)